Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/salvage.yml

```diff
@@ -6,6 +6,9 @@
     - type: NpcFactionMember
       factions:
         - Syndicate
+    - type: MindContainer
+      ghostOnShutdown: True
```

Comment:
Redundant, ghostOnShutdown is true by default.
https://github.com/DeltaV-Station/Delta-v/blob/df92f0f1f6b188cf2ffc80f0e760706442e0080a/Content.Shared/Mind/Components/MindContainerComponent.cs#L33-L39

Reply:
Good catch, thanks!